pod/perldata.pod

```diff
@@ -606,7 +606,11 @@ X<line> X<file> X<package>
 
 The two control characters ^D and ^Z, and the tokens __END__ and __DATA__
 may be used to indicate the logical end of the script before the actual
-end of file.  Any following text is ignored.
+end of file.  Any following text is ignored, except with __DATA__:
```

Comment:
This is not really true; the following text is still ignored by the interpreter, reading from DATA is a separate action that happens to reuse the source handle.

Reply:
And also it's not specific to `__DATA__`; the DATA handle will also recognize `__END__` in the top level script (as explained later in the section).

Reply:
Such text is also "not ignored" by POD parsers, but is still ignored by the interpreter, so I believe that's what this sentence is referring to. Perhaps just adding "by the Perl interpreter" or something to that effect would be better.